improving tools readme (#20305)
commaai/openpilot: tools/README.md

openpilot tools
============

System requirements
============

openpilot is developed and tested on **Ubuntu 20.04**, which is the primary development target aside from the [supported embdedded hardware](https://github.com/commaai/openpilot#supported-hardware). We also have a CI test to verify that openpilot builds on macOS, but the tools are untested. For the best experience, stick to Ubuntu 20.04, otherwise openpilot and the tools should work with minimal to no modifications on macOS and other Linux systems.

Setup
============
1. Clone openpilot into home directory:
```
cd ~

git clone --recurse-submodule https://github.com/commaai/openpilot.git
```

2. Run setup script:
Ubuntu:
```
source ~/openpilot/tools/ubuntu_setup.sh
```
MacOS:
```
source ~/openpilot/tools/mac_setup.sh
```

3. Compile openpilot by running SCons in openpilot directory
```

> scons --directory=openpilot

Reviewer: ```suggestion
cd openpilot && scons -j$(nproc)
```

openpilot tools
============

System requirements
============

> openpilot tools and the following setup steps are developed and tested on Ubuntu 20.04
> 
> We also test building openpilot on MacOS 10.14.2, but building is all that we can guarantee

Reviewer: ```suggestion
openpilot is developed and tested on *Ubuntu 20.04*, which is the primary development target aside from the [supported embdedded hardware](../#supported-hardware). We also have a CI test to verify that openpilot builds on macOS.
```

openpilot tools
============

System requirements
============

> openpilot tools and the following setup steps are developed and tested on:

Reviewer: elaborate on this whole section and the setup section. this is the primary place we should link to for getting openpilot setup on your PC for development

============

<!--ts-->

>  * [System requirements](#requirements)

Reviewer: remove table of contents

 * [Setup](#setup)
 * [Tool examples](#tool-examples)
   * [Replay driving data](#replay-driving-data)
   * [Debug car controls](#debug-car-controls)
   * [Juggle plots](#juggle-plots)
   * [Run openpilot in a simulator](#run-openpilot-in-a-simulator)
 * [Welcomed contributions](#welcomed-contributions)
<!--te-->

System requirements
============

openpilot tools and the following setup steps are developed and tested on:
* Ubuntu 20.04
* MacOS 10.14.2 (only building is tested on MacOS)

> * Python 3.8.2

Reviewer: remove this

============

> [Replay driving data](replay)

Reviewer: ```suggestion
[Replay a drive](replay)
```

-------------

> Review video and log data from routes and stream CAN messages to your EON.

Reviewer: ```suggestion
Review video and log data from routes and stream CAN messages to your device.
```

[Debug car controls](carcontrols)
-------------

Use a joystick to control your car.

> [Juggle plots](plotjuggler)

Reviewer: juggle doesn't mean anything to most people
